index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title> David Portfolio </title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <div> <img src="images/Portfolio_1.png" class="img_2"> </div>
+        <section class="section_1">
+         <h1> Tonic </h1>
+         <ul>
+     <li class="list_1"> CANOPY </li>
+     <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+     <li class="list3"><img src="images/Counter_2.png">2015</li>
+         </ul>
+         <p>A daily selection of privately 
+           personalized reads; no accounts 
+           or sign-ups required. </p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+     </section>
+        
+     <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_2.png" class="img_2"> </div>
+        <section class="section_1">
```

Comment:
- [ ] This section element here is out of place in terms of HTML semantics. Please remove it or replace it with `div`.